src/main/java/domain/GamePlayer.java

```diff
@@ -0,0 +1,48 @@
+package domain;
+
+import exception.DuplicateNumberException;
+import exception.InvalidNumberRangeException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class GamePlayer {
+
+    public static final String VALID_NUMBERS_RANGE = "[1-9]{3}";
+    public static final int VALID_NUMBERS_LENGTH = 3;
+    public static final String SPLIT_REGEX = "";
+    private final List<Integer> baseballNumbers;
+
+    public GamePlayer() {
+        baseballNumbers = null;
+    }
+
+    public GamePlayer(String numbers) {
+        if (!validateNumberRange(numbers)) throw new InvalidNumberRangeException();
+        if (!validateDuplicate(numbers)) throw new DuplicateNumberException();
```

Comment:
if문 안에 문장이 하나여도, 중괄호를 사용하는 것이 Java 컨벤션에서 권장하고 있는 사항입니다.

```suggestion
        if (!validateNumberRange(numbers)) {
            throw new InvalidNumberRangeException();
        }
        if (!validateDuplicate(numbers)) {
            throw new DuplicateNumberException();
        }
```

Reply:
네! 컨벤션에 더욱 주의 하겠습니다